src/Core/Core.csproj

```diff
@@ -4,7 +4,15 @@
     <ProjectName>Cythral.CloudFormation.CustomResource.Core</ProjectName>
     <AssemblyName>$(ProjectName)</AssemblyName>
     <PackageId>$(ProjectName)</PackageId>
-  </PropertyGroup>
+    <PublishRepositoryUrl>true</PublishRepositoryUrl>
```

Comment:
PublishRepositoryUrl is already set to true in the [Directory.Build.props file](https://github.com/cythral/cfn-custom-resource/blob/master/Directory.Build.props)